packages/terra-framework-docs/src/terra-dev-site/test/compact-interactive-list/CellContent.test.jsx

```diff
@@ -34,7 +36,56 @@ const FeaturedIcon = () => {
   const [isFeatured, setIsFeatured] = useState(false);
   const onButtonClick = () => setIsFeatured(!isFeatured);
   return (
-    isFeatured ? <Button variant="utility" text="Unfavorite item" icon={<IconFeatured />} onClick={onButtonClick} /> : <Button variant="utility" text="Favorite item" icon={<IconFeaturedOff />} onClick={onButtonClick} />
+    isFeatured
+      ? <Button variant="utility" text="Unfavorite item" icon={<IconFeatured />} onClick={onButtonClick} />
+      : <Button variant="utility" text="Favorite item" icon={<IconFeaturedOff />} onClick={onButtonClick} />
+  );
+};
+
+const PopupWithInteractiveContent = () => {
+  const [isOpen, setIsOpen] = useState(false);
+  const [selected, setSelected] = useState(null);
+  const buttonRef = useRef(null);
+
+  const setButtonNode = (node) => { buttonRef.current = node; };
+  const getButtonNode = () => buttonRef.current;
+  const handleButtonClick = () => { setIsOpen(true); };
+  const handleRequestClose = () => { setIsOpen(false); };
```

Comment:
These should have a useCallback hook.

Reply:
Callback added: https://github.com/cerner/terra-framework/pull/2149/commits/cacc605c3c50d58642b90fe8cef6d04dec530c9c